webapp/views/index.hbs

```diff
@@ -9,23 +9,68 @@
   </head>
   <body>
     <main>
-      {{> HUD}}
-      <div class="board">
+      <aside class="players-hud">
+        <div class="turn-counter">
+            <h6>Turn</h6>
+            <em>{{game.state.turnNumber}}</em>
+        </div>
+        <div class="players">
+            <div class="black">
+                <i class="fa fa-user-alt"></i>
+                <div class="hexagon-container">
+                    {{#contains game.state.currentPlayer "black" }}
+                        <em class="hexagon-text--dark">{{game.state.remainingTiles.black}}</em>
+                        <div class="hexagon-icon hexagon-icon--gold"></div>
+                    {{else}}
+                        <em class="hexagon-text">{{game.state.remainingTiles.black}}</em>
+                        <div class="hexagon-icon"></div>
+                    {{/contains}}
+                </div>
+            </div>
+            <div class="white">
+                <i class="fa fa-user-alt"></i>
+                <div class="hexagon-container">
+                    {{#contains game.state.currentPlayer "white" }}
+                        <em class="hexagon-text--dark">{{game.state.remainingTiles.white}}</em>
+                        <div class="hexagon-icon hexagon-icon--gold"></div>
+                    {{else}}
+                        <div class="hexagon-icon"></div>
```

Comment:
Seems like it's missing
`<em class="hexagon-text">{{game.state.remainingTiles.white}}</em>`